changes/add_nabuSignoff.md

```diff
@@ -0,0 +1,3 @@
+Ability to sign off cases for Analysis Review, Release Approval, and Release directly from Dimsum.
+Note that sign-offs are saved in Nabu and it will take some time for them to show up in Dimsum after
```

Comment:
This delay could lead to the situation where one person signs off, the signoff does not appear, then someone else submits a second duplicate signoff, couldn't it? What happens in that situation?

Reply:
The second sign-off would overwrite the first. I am currently working on caching these in Dimsum so there is no delay